tests/dogtag/pytest-ansible/provision/osp_provision.py

```diff
@@ -0,0 +1,192 @@
+import os
+import random
+import sys
+
+import click
+import openstack
+
+import osp_vars as constants
+
+OS_NETWORK_ID = random.choice(constants.OS_NETWORK_ID)
+
+arch = "x86_64"
+nightly_latest = "nightly-latest"
+production_latest = 'production-latest'
+inventory_file = "ansible_inventory"
+instance_record = '/{}/instance_record.txt'.format(os.getcwd())
+
+
+def create_connection(auth_url, region, project_name, username, password):
+    return openstack.connect(auth_url=auth_url,
+                             project_name=project_name,
+                             username=username,
+                             password=password,
+                             user_domain_name=constants.DOMAIN_NAME,
+                             project_domain_name=constants.DOMAIN_NAME,
+                             region_name=region,
+                             app_name='RHCS_QE',
```

Comment:
Could we get the value from environment variable?

Reply:
Done changes